### PR TITLE
add antifreeze manifest checking

### PR DESCRIPTION
**Pull request by xoebus**
This uses the [`antifreeze` plugin](https://github.com/odlp/antifreeze) to check that there are no unexpected services or environment variables that will be destroyed by the deploy.

This requires minor changes to the main pipeline in order to add a GitHub release resource for `antifreeze`. I didn't want to do that though until this got a :+1:.

### CODE REVIEW

File: out/cloud_foundry.go [hunk 1]
```diff
@@ -36,12 +36,21 @@ func (cf *CloudFoundry) Target(organization string, space string) error {
 }
 
 func (cf *CloudFoundry) PushApp(manifest string, path string, currentAppName string) error {
+	zeroDowntimeDeploy := currentAppName != ""
+
+	if zeroDowntimeDeploy {
+		err := cf.cf("check-manifest", currentAppName, "-f ", manifest).Run()
```
Comment (vito) [inline out/cloud_foundry.go]: `"-f "` should be `"-f"`

Comment (xoebus) [inline out/cloud_foundry.go]: Removed.

Funny that the tests pass both with and without it. Something must be stripping them at a lower level.

**Comment by pivotal-issuemaster** [comment]
@xoebus Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=concourse/cf-resource&pullRequestId=19) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/faq) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@xoebus Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/cf-resource&pullRequestId=19)!

**Comment by chendrix** [comment]
Hey @xoebus is this something you even still want?

**Comment by chendrix** [comment]
After reviewing, it looks like this isn't something we want.

**Pull request closed without merging**